### PR TITLE
fix result type, fixup responses from the ui to api

### PR DESCRIPTION
**Pull request by Hellowlol**
todo unify json responses from webserve.py

### CODE REVIEW

File: plexpy/api2.py [hunk 1]
```diff
@@ -524,13 +506,13 @@ def _api_run(self, *args, **kwargs):
...
-            if self._api_debug:
+            try:
                 result = call(**self._api_kwargs)
-            else:
-                try:
-                    result = call(**self._api_kwargs)
-                except Exception as e:
-                    logger.error(u'PlexPy APIv2 :: Failed to run %s %s %s' % (self._api_cmd, self._api_kwargs, e))
+            except Exception as e:
+                if self._api_debug:  # check this with j
```
Comment (Hellowlol) [inline plexpy/api2.py]: @JonnyWong16 Whats your take this on? I would like to get the traceback in the browser if the debug param is used with the api.

Comment (JonnyWong16) [reply plexpy/api2.py]: Fine with me, only if `debug=1`.

File: plexpy/webserve.py [hunk 2]
```diff
@@ -329,6 +329,7 @@ def get_recently_added(self, count='0', **kwargs):
     @cherrypy.expose
     @cherrypy.tools.json_out()
     @requireAuth(member_of("admin"))
+    @addtoapi()
     def delete_temp_sessions(self, **kwargs):
```
Comment (JonnyWong16) [inline plexpy/webserve.py:333]: Needs a docstring.

```
Flush out all of the temporary sessions in the database.
```

And run `docs_md` to  update the API docs.

**Comment by Hellowlol** [comment]
Should be good now.

**Pull request merged**